### PR TITLE
Added bandit to tox test

### PR DESCRIPTION
**Pull request by qduk**
I also went through the code and put exceptions where they seemed applicable. I told bandit to skip [B101](https://bandit.readthedocs.io/en/latest/plugins/b101_assert_used.html) as it didn't apply to us.

### CODE REVIEW

File: .bandit.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+---
+skips: ["B101"]
```
Comment (jmcgill298) [inline .bandit.yml:2]:
```suggestion
skips: ["B101"]

```

File: pyntc/devices/aireos_device.py [hunk 2]
```diff
@@ -63,7 +63,7 @@ class AIREOSDevice(BaseDevice):
 
     vendor = "cisco"
 
-    def __init__(self, host, username, password, secret="", port=22, **kwargs):
+    def __init__(self, host, username, password, secret="", port=22, **kwargs):  # nosec
```
Comment (jmcgill298) [inline pyntc/devices/aireos_device.py:66]: what is the security issue being reported?

File: tox.ini [hunk 3]
```diff
@@ -46,4 +51,4 @@ exclude =
     .pytest_cache
 
 # line length is handled by black
-ignore = E501
+ignore = E501
```
Comment (jmcgill298) [inline tox.ini:54]:
```suggestion
ignore = E501

```

File: pyntc/devices/aireos_device.py (hunk 2 above)
Comment (jmcgill298) [reply pyntc/devices/aireos_device.py:66]: I think this is B107, which we should just ignore in the bandit config file

File: .bandit.yml (hunk 1 above)
Comment (qduk) [reply .bandit.yml]: Updated.

File: pyntc/devices/aireos_device.py (hunk 2 above)
Comment (qduk) [reply pyntc/devices/aireos_device.py]: That is correct. Added B107 to exclude list on bandit config file.

File: tox.ini (hunk 3 above)
Comment (qduk) [reply tox.ini]: Updated

File: pyntc/devices/aireos_device.py (hunk 2 above)
Comment (jmcgill298) [reply pyntc/devices/aireos_device.py]: Sorry, I meant that when adding that to the exclude list, you would remove the `# nosec` comments from the corresponding function defs

File: pyntc/devices/f5_device.py [hunk 4]
```diff
@@ -285,7 +285,9 @@ def _upload_image(self, image_filepath):
                     end = size
                 content_range = "{}-{}/{}".format(start, end - 1, size)
                 headers["Content-Range"] = content_range
-                requests.post(_URI, auth=(self.username, self.password), data=payload, headers=headers, verify=False)
+                requests.post(
+                    _URI, auth=(self.username, self.password), data=payload, headers=headers, verify=False  # nosec
+                )
```
Comment (qduk) [inline pyntc/devices/f5_device.py:290]: This is bandit test B501.

File: pyntc/devices/aireos_device.py (hunk 2 above)
Comment (qduk) [reply pyntc/devices/aireos_device.py]: I did that. The function with the `nosec` still is bandit test B501.

**Review by jmcgill298 — APPROVED** [review]

**Pull request merged**